src/components/PersonLink.tsx

```diff
@@ -0,0 +1,84 @@
+import { Link, useLocation, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+import { GetPersonDates } from '../services/getPersonDates';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, born, died, slug, motherName, fatherName } = person;
+
+  const MOTHER_NAME = motherName;
+  const FATHER_NAME = fatherName;
```

Comment:
The constants `MOTHER_NAME` and `FATHER_NAME` are redundant since they are directly assigned from `person.motherName` and `person.fatherName`. You can use `person.motherName` and `person.fatherName` directly in `GetPersonDates` calls.